samples/ts-valueentity-shopping-cart/test/testkit.ts

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Mocks the behaviour of a single Akka Serverless EventSourcedEntity.
+ *
+ * Handles any commands and events, internally tracking the state and maintaining an event log.
+ *
+ * NOTE: Entity IDs are not handled, so all commands are assumed to refer to a single entity.
+ */
+export class MockEventSourcedEntity {
```

Comment:
Looks like this `testkit` should be part of the SDK rather than the code generation? Add an issue to follow up on this?

Reply:
Good idea, issue created: https://github.com/lightbend/akkaserverless-javascript-sdk/issues/168